index.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+const axios = require('axios');
+const path = require('path');
+const fs = require('fs-extra');
+const program = require('commander');
+
+let input;
+
+program
+    .name('icaredata-client')
+    .usage('<path-to-messages> [options]')
+    .option('-o, --out <out>', 'the path to the output folder', path.join('.', 'out'))
+    .arguments('<path-to-messages>')
+    .action(function (pathToMessages) {
+        input = pathToMessages;
+    })
+    .parse(process.argv);
```

Comment:
I had no idea this existed, it's pretty cool!

Reply:
Yeah, commander is a cool tool!